app/components/ViewContainer.js

```diff
@@ -47,15 +48,16 @@ export default class ViewContainer extends Component {
         // <AR currLoc={ this.state.currLoc } pins= { pins } />
 
   render() {
-    const {getLocationToSave, pins} = this.props;
+    const {getLocationToSave, pins, recent} = this.props;
```

Comment:
The spacing between the curly braces are inconsistent. Should we pick one and stick to it?